tensorflow/python/autograph/converters/variables_test.py

```diff
@@ -124,7 +124,7 @@ def f(a, b, c):
 
   def test_del_item_multiple_mixed_used_after(self):
 
-    def f(a, b, c):
+    def f(a, b, c):   # TODO: Unused parameters 'a', 'b'
```

Comment:
These are intentional as well. Note the explicit `del` forces a "use" to signal that.